### ISSUE TITLE
alpha.3 is gone?

### ISSUE THREAD

**Issue by gattytto** [LibreTime/libretime#295]
I set up in lxc with alpha.3 and now trying in a new container only alpha.1 and alpha.2 are available..

**Comment by hairmare** [comment]
alpha.3 hasn't been released yet. I plan on releasing it soonish but was hoping that I would be able to add debian-9 support before.

**Comment by stanlog** [comment]
Hello @hairmare I'm really interested to update my Libretime instance under Stretch (I already use Libretime 3.0.0 alpha 1 under a Jessie VM). I read the install and installer/lib/requirements-debian-jessie.apt files, is there something else to customize for trying the stretch install with libretime 3.0.0 alpha 2 ? Thanks a lot for your work.

**Comment by hairmare** [comment]
@stanlog I think the files you mention basically cover the needed changes. I started implementing debian-9 support a while ago with https://github.com/LibreTime/libretime/compare/master...radiorabe:feature/debian-9-bump but never got around to making the PR since I got side tracked with switching my vagrant setup to use libvirt rather than vbox for performance reasons. The vagrant setup is important to make it easier for developers to reproduce any stretch specific issues.
